knpu/episode3/protected-visibility.md

```diff
@@ -12,7 +12,9 @@ function and paste it over here. And then just add '(rebel)' at the end. Now
 you may be thinking "guys, that's some serious code duplication...". Well you're
 absolutely right, and we'll get to fixing that!
 
-For now what we've got is pretty straightforward, so let's refresh and ....oh
+[[[ code('7db3c66c91') ]]]
```

Comment:
I like to include the code blocks as early as possible (code block first, explanation after). So, I'd move this up, right after `And then just add '(rebel)' at the end.`